ci-scripts/test_agent/main.py

```diff
@@ -23,32 +23,46 @@ class WorkerState:
 
 
 num_of_testers = 1
+# Initialize firebase client as global
+db_client = FirebaseClient()
 
 
-def test_done_callback(tester_id, workload, valid, report):
+def test_done_callback(tester_id, workload, verdict, report):
     db_client.mark_workload_done(workload)
-    db_client.push_test_report(workload, valid, report)
+    db_client.push_test_report(workload, verdict, report)
     return
 
 
 def main():
-    # Initialize firebase client
-    db_client = FirebaseClient()
     state = WorkerState.READY
     testers = []
     for i in range(num_of_testers):
         testers.append(Tester(i))
-
     while True:
         testers_state = []
         for tester in testers:
             if tester.is_ready():
+                print("tester is ready")
```

Comment:
maybe print which test this is. In case we have more than 1. add tester.get_id()